flink-ml-core/src/main/java/org/apache/flink/ml/linalg/DenseVector.java

```diff
@@ -51,6 +51,35 @@ public double[] toArray() {
         return values;
     }
 
+    @Override
+    public DenseVector toDense() {
+        return this;
+    }
+
+    @Override
+    public SparseVector toSparse() {
+        int nnz = 0;
```

Comment:
I am not sure `nnz` is an intuitive name. The general rule is to make variable names either self-explanatory or simple.

Could we change this to either `non_zero_count` or `count`?

Same for `ii` and `vv`.